keyboards/cannonkeys/cerberus/solderable/keyboard.json

```diff
@@ -0,0 +1,711 @@
+{
+    "keyboard_name": "Cerberus",
+    "backlight": {
+        "breathing": true,
+        "levels": 6,
+        "pin": "A6"
+    },
+    "features": {
+        "backlight": true
+    },
```

Comment:
Backlight config but no keycodes

Reply:
Just took care of this - thanks again for the catch.

Ready for review again.